docs/canarytokens/actions.md

```diff
@@ -225,6 +225,68 @@ These are a collection of endpoints that allow you mint new, interact with, and
 
 <APIDetails :endpoint="$page.frontmatter.endpoints.create"/>
 
+#### Example
+
+
+:::: tabs :options="{ useUrlFragment: false }"
+
+::: tab "cURL"
+
+``` bash
+curl https://EXAMPLE.canary.tools/api/v1/canarytoken/create \
+  -d auth_token=EXAMPLE_AUTH_TOKEN \
+  -d memo=EXAMPLE_MEMO \
```

Comment:
Here you use `EXAMPLE_MEMO, in the python example you use `EXAMPLE_TOKEN_MEMO` and in the response you have `Example Memo`. Let's be consistent here: `'Example Memo'` works.